renovate.json

```
@@ -1,8 +1,9 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["github>netlify/renovate-config:esm"],
-  "ignorePresets": [":prHourlyLimit2"],
-  "semanticCommits": "enabled",
+  "automerge": true,
   "dependencyDashboard": true,
-  "automerge": true
+  "ignorePresets": [":prHourlyLimit2"],
+  "ignorePaths": ["**/benchmarks/fixtures", "**/tests/fixtures"],
```

Looking through patterns in this monorepo, I'm seeing `tests/fixtures`, `test/fixtures`, `test/foo/bar/fixtures`, etc. So maybe something like this to capture all of them:
```suggestion
  "ignorePaths": ["**/fixtures"],
```